crypto/src/hash/rescue_prime/parameters.rs

```diff
@@ -0,0 +1,416 @@
+    Fp::const_from_raw(3932918848577657311),
+    Fp::const_from_raw(16327200574281469287),
+    Fp::const_from_raw(7060041503065075033),
+    Fp::const_from_raw(4892761442718320741),
+    Fp::const_from_raw(8255275116206368067),
+    Fp::const_from_raw(14957838536671021552),
+    Fp::const_from_raw(14493715972468567436),
+    Fp::const_from_raw(7463718209809697261),
+    Fp::const_from_raw(3440982266989812843),
+    Fp::const_from_raw(2354199421703013492),
+    Fp::const_from_raw(2321628279578256047),
+    Fp::const_from_raw(3746041501354899488),
+    Fp::const_from_raw(11186576936873825301),
+    Fp::const_from_raw(15218587616061641074),
+    Fp::const_from_raw(11844784525417523222),
+    Fp::const_from_raw(7998727848169056055),
+    Fp::const_from_raw(7948968711630609066),
+    Fp::const_from_raw(11805042600408037937),
+    Fp::const_from_raw(18172588443872800894),
+    Fp::const_from_raw(13092373363317372568),
+    Fp::const_from_raw(2169983441195298580),
+    Fp::const_from_raw(1499680808057735775),
+    Fp::const_from_raw(7077486803310915643),
+    Fp::const_from_raw(743612288630452727),
+    Fp::const_from_raw(11665426394426065172),
+    Fp::const_from_raw(15533499373769144802),
+    Fp::const_from_raw(14249183160150274240),
+    Fp::const_from_raw(13792290235996127743),
+    Fp::const_from_raw(4995017088228886738),
+    Fp::const_from_raw(9763845271226970122),
+    Fp::const_from_raw(1727820159257625458),
+    Fp::const_from_raw(9681902124347643227),
+    Fp::const_from_raw(11327574568051933160),
+    Fp::const_from_raw(10627429556158481577),
+    Fp::const_from_raw(13984143774797145216),
+    Fp::const_from_raw(17082059622058840713),
+    Fp::const_from_raw(16264233536802058333),
+    Fp::const_from_raw(10077962488096645822),
+    Fp::const_from_raw(5057253598123536060),
+    Fp::const_from_raw(2301672207952647376),
+    Fp::const_from_raw(17506877517896521554),
+    Fp::const_from_raw(14583366393971011156),
+    Fp::const_from_raw(6226877164823354372),
+    Fp::const_from_raw(2260055134098203623),
+    Fp::const_from_raw(12945296184826522120),
+    Fp::const_from_raw(15417698598606677168),
+    Fp::const_from_raw(7447949755934804788),
+    Fp::const_from_raw(8017843736725863212),
+    Fp::const_from_raw(1003688007091182795),
+    Fp::const_from_raw(8935767355090348282),
+    Fp::const_from_raw(793319158990348431),
+    Fp::const_from_raw(4437923789992338287),
+    Fp::const_from_raw(7869978205237541489),
+    Fp::const_from_raw(9039403419111053092),
+    Fp::const_from_raw(3845065612997771849),
+    Fp::const_from_raw(15179573672801872590),
+    Fp::const_from_raw(2879645310341005490),
+    Fp::const_from_raw(4421001170561580576),
+    Fp::const_from_raw(7614461260369642079),
+    Fp::const_from_raw(10869617590371203777),
+    Fp::const_from_raw(4582902440098948914),
+];
+
+pub enum SecurityLevel {
+    Sec128,
+    Sec160,
+}
+
+pub fn get_round_constants(level: SecurityLevel) -> &'static [Fp] {
```

Comment:
Why return `&'static [Fp]` instead of `[Fp]`?

Reply:
Using `[Fp] `leads to compile error since we return arrays of two different sizes.  ` &'static [Fp]` seems to solve it

Reply:
&'static is correct here since the function is literally returning references to static (a.k.a global) arrays